src/power_grid_model/core/buffer_handling.py

```diff
@@ -245,7 +245,7 @@ def _get_sparse_buffer_properties(
             if attribute_data.ndim != ndim + schema.dtype[attribute].ndim or attribute_data.shape[:ndim] != shape:
                 raise ValueError(f"Data buffers must be consistent. {VALIDATOR_MSG}")
 
-    contents_size = sum(shape)
+    contents_size = shape[0]
```

Comment:
i actually wonder whether this is correct. the original one wasn't, but this is the full size of the buffer, right?

Reply:
ok, good one. i hereby also approve